src/SdkGenerator/SdkGenerator.csproj

```diff
@@ -1,74 +1,85 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-    <PropertyGroup>
-        <OutputType>Exe</OutputType>
-        <TargetFramework>net7.0</TargetFramework>
-        <PackAsTool>true</PackAsTool>
-        <ToolCommandName>SdkGenerator</ToolCommandName>
-        <PackageOutputPath>../../</PackageOutputPath>
-    </PropertyGroup>
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>SdkGenerator</ToolCommandName>
+    <PackageReadmeFile>docs/README.md</PackageReadmeFile>
+    <Description>A command line dotnet tool to retrieve swagger (OpenAPI) files and generate
+      easy-to-use markdown documentation or friendly software development kit libraries. Can be
+      automated using Github Workflows.</Description>
+    <PackageTags>SDK generator swagger openapi swashbuckle</PackageTags>
+    <Copyright>Copyright 2021 - 2023</Copyright>
+    <PackageReleaseNotes># 1.1.6
+      August 21, 2023
+
+      * Fixed capitalization of DotnetToolSettings.xml
+    </PackageReleaseNotes>
+    <PackageIcon>docs/icons-puzzle.png</PackageIcon>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
```

Comment:
Using a license expression instead of a file is recommended these days, because there's a lot of reporting/tooling on License Expressions (they are standardized) that is much harder with files.

Reply:
Good idea! 

I’m pretty sure I still need to have a license file to cite other sources I used, like the open source icon and the original location for other MIT code. I’ll update that as well.